packages/bpk-component-nudger/README.md

```diff
@@ -129,22 +127,25 @@ class App extends Component {
 
 ### BpkConfigurableNudger
```

Comment:
Should we have 2 prop tables? 
1 for one with label, one for without? 

This way we can show the title is required etc.

Reply:
I'm afraid that'd make it look like there's 2 separate components (as that is the only time when we have more than 1 api table) and that's not the case here, but maybe can add a comment in the api table that title is required if you want to show a label?